components/renku_data_services/session/api.spec.yaml

```diff
@@ -111,93 +219,218 @@ paths:
+      responses:
+        "201":
+          description: The started session
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Session"
+        default:
+          $ref: "#/components/responses/Error"
       tags:
         - sessions
+  /projects/{project_id}/session_launchers:
+    get:
+      summary: Get a project's session launchers
+      parameters:
+        - in: path
+          name: project_id
+          required: true
+          schema:
+            type: string
+      responses:
+        "200":
+          description: List of sessions launchers
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/LaunchersList"
+        default:
+          $ref: "#/components/responses/Error"
+      tags:
+        - session_launchers
 components:
   schemas:
-    SessionsList:
-      description: A list of Renku sessions
+    EnvironmentList:
+      description: A list of session environments
       type: array
       items:
-        $ref: "#/components/schemas/Session"
+        $ref: "#/components/schemas/Environment"
+    Environment:
+      description: A Renku 1.0 session environment
+      type: object
+      properties:
+        id:
+          $ref: "#/components/schemas/Ulid"
+        name:
+          $ref: "#/components/schemas/Name"
+        creation_date:
+          $ref: "#/components/schemas/CreationDate"
+        description:
+          $ref: "#/components/schemas/Description"
+        container_image:
+          $ref: "#/components/schemas/ContainerImage"
+      required:
+        - id
+        - name
+        - creation_date
+        - container_image
+      example:
+        id: 01AN4Z79ZS6XX96588FDX0H099
+        name: JupyterLab environment
+        creation_date: "2023-11-01T17:32:28Z"
+        description: JupyterLab session environment
+        container_image: renku-jupyter:latest
+    EnvironmentPost:
+      description: Data required to create a session environment
+      type: object
+      properties:
+        name:
+          $ref: "#/components/schemas/Name"
+        description:
+          $ref: "#/components/schemas/Description"
+        container_image:
+          $ref: "#/components/schemas/ContainerImage"
+      required:
+        - name
+        - container_image
+    EnvironmentPatch:
+      type: object
+      description: Update a session environment
+      additionalProperties: false
+      properties:
+        name:
+          $ref: "#/components/schemas/Name"
+        description:
+          $ref: "#/components/schemas/Description"
+        container_image:
+          $ref: "#/components/schemas/ContainerImage"
+    LaunchersList:
+      description: A list of Renku session launchers
+      type: array
+      items:
+        $ref: "#/components/schemas/Launcher"
       minItems: 0
-    Session:
-      description: A Renku native session definition and metadata
+    Launcher:
```

Comment:
Should we stick to `SessionLauncher`?

Reply:
Done in [36de683](https://github.com/SwissDataScienceCenter/renku-data-services/pull/122/commits/36de683e53357bc8674ae4e82ec3b8ceed8803cd).